turbopack/crates/turbo-tasks/src/task/task_input.rs

```diff
@@ -278,14 +283,36 @@ where
 
     fn is_resolved(&self) -> bool {
         self.iter()
-            .all(|(k, v)| TaskInput::is_resolved(k) || TaskInput::is_resolved(v))
+            .all(|(k, v)| TaskInput::is_resolved(k) && TaskInput::is_resolved(v))
```

Comment:
I noticed this condition was wrong, but we don't use `BTreeMap` in many places, which is probably why we didn't notice it.